docker/test/integration/runner/compose/docker_compose_mongo.yml

```diff
@@ -9,3 +9,9 @@ services:
         ports:
             - ${MONGO_EXTERNAL_PORT}:${MONGO_INTERNAL_PORT}
         command: --profile=2 --verbose
+
+    mongo2:
+        image: mongo:latest
```

Comment:
It is better to use fixed version,
otherwise our CI will depend on random changes when Docker images will be rebuilt.